platform/linux-generic/odp_ipsec.c

```diff
@@ -953,13 +953,14 @@ static int ipsec_out_tunnel_parse_ipv6(ipsec_state_t *state,
 				       ipsec_sa_t *ipsec_sa)
 {
 	_odp_ipv6hdr_t *ipv6hdr = state->ip;
+	uint32_t ver_tc_flow = odp_cpu_to_be_32(ipv6hdr->ver_tc_flow);
```

Comment:
This should use odp_be_to_cpu_32(). Even though the effect is the same, that is the logical direction of the conversion here.

Reply:
Will update in next version